lib/migration.js

```diff
@@ -701,6 +701,12 @@ function mixinMigration(MySQL, mysql) {
     var p = this.getModelDefinition(model).properties[prop];
     var line = this.columnDataType(model, prop) + ' ' +
       (this.isNullable(p) ? 'NULL' : 'NOT NULL');
+    if (typeof p.mysql !== 'undefined' && p.mysql.default && typeof p.mysql.default === 'string') {
+      line = line + ' DEFAULT ' +
+            (p.mysql.default.toUpperCase() === 'CURRENT_TIMESTAMP' || p.mysql.default.toLowerCase() === 'now' ?
+              'CURRENT_TIMESTAMP' : '"' + p.mysql.default + '"');
```

Comment:
> It also says these types aren't supported:
`BLOB, TEXT, and JSON columns cannot be assigned a default value.`

> I can add logic to prevent these types from attempting to set a default.

Yes please.

Also my original concern was that we stringify all values. What if I have an integer column and want my default value to be 10? The current code will always insert it as `"10"` and throw error.

Can we also add properties of each data type with default values and add test cases for it?